docs/reference/bidi.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BiDi Protocol Support
+---
+
+XCUITest driver has partial support of the [BiDi Protocol](https://w3c.github.io/webdriver-bidi/) since version 7.26.0.
+Only events and commands mentioned below are supported.
+All other entities described in the spec throw not implemented errors.
+
+# Supported Events
+
+## log.entryAdded
+
+This event is emitted if the driver retrieves a new entry for any of the below log types. Logs collection might be disabled by the `appium:skipLogCapture` capability.
+
+### crashlog
+
+Events are emitted for both emulator and real devices. The latter only works if [py-ios-device](hhttps://github.com/YueChen-C/py-ios-device) is installed on the server host. Each event contains a particular device crash report entry.
+Events are always emitted with the `NATIVE_APP` context.
+
+### syslog
+
+Events are emitted for both emulator and real devices. Each event contains a single device system log line.
+Events are always emitted with the `NATIVE_APP` context.
+
+### safariConsole
+
+Events are emitted for both emulator and real devices. Each event contains a single safari console log line.
```

Comment:
I guess Safari should be capitalised (same below)

Reply:
fixed